compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/Patterns.cpp

```diff
@@ -247,6 +247,10 @@ struct ConvertTensorDialectReshapeOpPattern
       Value element = rewriter.create<tensor::ExtractOp>(loc, op.getShape(),
                                                          ValueRange({idx}));
       if (ShapedType::isDynamic(resultType.getShape()[i])) {
+        auto elementTy = element.getType();
+        if (isa<IntegerType>(elementTy))
+          element = rewriter.create<arith::IndexCastOp>(
+              loc, rewriter.getIndexType(), element);
```

Comment:
style nit: IREE style requires to wrap single-statement with braces. This is different from MLIR repo.

```suggestion
        if (isa<IntegerType>(elementTy)) {
          element = rewriter.create<arith::IndexCastOp>(
              loc, rewriter.getIndexType(), element);
         }
```